### DISCUSSION TITLE
From this new update css blocked over HTTPS

### DISCUSSION THREAD

**Discussion by AtizaD** [Stirling-Tools/Stirling-PDF#1239] [Q&A] [answered]
Mixed Content: The page at 'https://mydomain/' was loaded over HTTPS, but requested an insecure favicon 'http://mydomain/favicon.ico?'. This request has been blocked; the content must be served over HTTPS.

In this new update CSS is blocked over HTTPS. Any solution?

**Comment by Frooodle** [comment]
someone else reporting the same and said it was fixed by " using portmapping and not pointing into the container directly"

Not sure what caused this issue in latest update still looking into it

**Reply by AtizaD** [reply]
okay thank you

**Reply by Frooodle** [reply]
Let me know if the same fix works for you!

**Reply by AtizaD** [reply]
It seems your 2 hours ago update is making it work now. I started the whole process again and it now works. No port mapping

**Reply by Frooodle** [reply]
No changes related to that for this..... very odd glad it works though

**Reply by AtizaD** [reply, marked as answer]
server {
    server_name mydomain;

    location / {
        proxy_pass http://localhost:8080;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
    }

    listen 443 ssl; # managed by Certbot
    ssl_certificate /etc/letsencrypt/live/mydomain/fullchain.pem; # managed by Certbot
    ssl_certificate_key /etc/letsencrypt/live/mydomain/privkey.pem; # managed by Certbot
    include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
    ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot

}

server {
    if ($host = mydomain) {
        return 301 https://$host$request_uri;
    } # managed by Certbot


    listen 80;
    server_name mydomain;
    return 404; # managed by Certbot


}


Am glad it works. Thank you guys
